src/CMakeLists.txt

```diff
@@ -81,3 +83,64 @@ if (RUBY_FOUND)
    	  --gtest_output=xml:${CMAKE_BINARY_DIR}/test_results/${test}rb.xml)
   endforeach()
 endif()
+
+#################################
+# Create and install Python interfaces
+# Example usage
+# $ export PYTHONPATH=/ws/install/lib/python/:$PYTHONPATH
+if (PYTHONLIBS_FOUND)
+  set_source_files_properties(python/python.i PROPERTIES CPLUSPLUS ON)
+  set_source_files_properties(python/python.i PROPERTIES SWIG_FLAGS "-includeall")
+  set_source_files_properties(python/python.i PROPERTIES SWIG_MODULE_NAME "math")
+  set(SWIG_PY_LIB pymath)
+  set(SWIG_PY_LIB_OUTPUT math)
+
+  set(CMAKE_SWIG_OUTDIR "${CMAKE_BINARY_DIR}/lib/python")
+  if(CMAKE_VERSION VERSION_GREATER 3.8.0)
+    SWIG_ADD_LIBRARY(${SWIG_PY_LIB} LANGUAGE python SOURCES python/python.i ${sources})
```

Comment:
We are recompiling sources instead of just linking, for both python and ruby bindings.

Reply:
Solved at 0ce31c6

Reply:
It fails now, it's like now it can't find `libignition-math6.so` but I couldn't replicate this locally, locally it works. Will check

Reply:
I can reproduce the problem on my machine. I'm looking into it